indexer/feature_utils.cpp

```diff
@@ -438,4 +448,19 @@ string GetLocalizedFeeType(TypesHolder const & types)
   return localized_types[0];
 }
 
+string GetLocalizedSecondaryAtm(TypesHolder const & types)
```

Comment:
Is this wrapper needed? Can't the string be translated by using "type.amenity.atm" key in iOS and Android?

Reply:
Removed